### PR TITLE
fix(linux): Fix launch of km-config

### PR DESCRIPTION
**Pull request by ermshiperete**
km-config provides a .desktop file that allows the user to start the configuration app like any other app. Unfortunately, a recent
change that improved the consistency of the manual pages broke that because the .desktop file used to call km-config with the parameter `-i` (plus file if given). With the recent change `-i` now requires a filename, which caused `km-config` to output an error
and quit. For the user this looked like it didn't do anything.

This change adds the file/url as a new optional positional parameter while keeping the existing `-i`/`--install` parameter. 

This PR also fixes the km-config man page. Previously the included file replaced the entire description, including the parameters. With this change the text from the include file gets added to the beginning of the description.

### CODE REVIEW

**Comment by ermshiperete** [comment]
# User testing

@keymanapp/testers

- [ ] (TEST-KMCONFIG-1): launch keyman configuration by typing Windows key, then "Keyman Keyboards". This should bring up the "Keyman Configuration" window.
- [ ] (TEST-KMCONFIG-2): verify man page: open a terminal window, type `man km-config`. The displayed man page should list a reasonable description, the optional parameters as well as the new positional parameter.

File: linux/keyman-config/km-config [hunk 1]
```diff
@@ -12,14 +12,23 @@ if __name__ == '__main__':
...
+                                     'keyboard packages from the website or install from ' +
+                                     'local files.')
     parser.add_argument('--version', action='version', version='%(prog)s version ' + versionstring)
     parser.add_argument('-v', '--verbose', action='store_true', help='verbose logging')
     parser.add_argument('-vv', '--veryverbose', action='store_true', help='very verbose logging')
     parser.add_argument('-i', '--install', action='store', help='download and/or install .kmp ' +
                         'package. INSTALL can either be a downloaded .kmp file, a file:// URL ' +
                         'pointing to a .kmp file, or a keyman:// URL, possibly with a ' +
                         'bcp47=<language> specified.')
+    parser.add_argument('url', nargs='?', default='', metavar='INSTALL',
```
Comment (darcywong00) [inline linux/keyman-config/km-config:27]: Would a command-line install look like this?
```
km-config url https://downloads.keyman.com/keyboards/sil_euro_latin/1.9.5/sil_euro_latin.kmp
```

Comment (ermshiperete) [reply linux/keyman-config/km-config:27]: It's simply:

```
km-config https://downloads.keyman.com/keyboards/sil_euro_latin/1.9.5/sil_euro_latin.kmp
```

**Comment by MakaraSok** [comment]
```
ubuntu@ubuntu:~$ km-config --version
km-config version 15.0.102-alpha
ubuntu@ubuntu:~$ dpkg -l | grep keyman
ii  ibus-keyman                                15.0.102-1+focal1                     amd64        Keyman engine for IBus
ii  keyman                                     15.0.102-1+focal1                     all          Type in your language with Keyman for Linux
ii  onboard-keyman                             1.4.1-5.0~focal                       amd64        Simple On-screen Keyboard
ii  onboard-keyman-common                      1.4.1-5.0~focal                       all          Simple On-screen Keyboard (common files)
ii  onboard-keyman-data                        1.4.1-5.0~focal                       all          Language model files for the word suggestion feature of Onboard
ii  python3-keyman-config                      15.0.102-1+focal1                     all          Keyman for Linux configuration
```

**Comment by ermshiperete** [comment]
@keymanapp-test-bot retest

**Comment by ermshiperete** [comment]
@MakaraSok Can you please try again with build 15.0.107-16 (follow link behind "Test: Keyman packaging (Linux) — Jenkins build succeeded" in GitHub checks section below)

**Comment by MakaraSok** [comment]
- **TEST_KMCONFIG_1: PASSED** Keyman Configuration does open
The Windows key doesn't open what can be typed to search for "Keyman Keyboards". What works is to click on the `Show Application` and then search for "Keyman Keyboards". Clicking on "Keyman Keyboards" shows the Keyman Configuration.
- **TEST_KMCONFIG_2: PASSED** The description does show up when running `man km-config`.
```
KM-CONFIG(1)                                                       User Commands                                                      KM-CONFIG(1)

NAME
       km-config - Launches Keyman Configuration for installing and showing information about Keyman keyboards

SYNOPSIS
       km-config
        [-h] [--version] [-v] [-vv] [-i INSTALL] [INSTALL]

DESCRIPTION
       Keyman  makes  it  possible for you to type in over 1,000 languages on Windows, Linux, macOS, iPhone, iPad, Android tablets and phones, and
       even instantly in your web browser.

       With the world's most powerful keyboarding engine, intuitive and rapid text input is now possible in your language, and for over 99% of the
       global population's mother tongues!

       km-config  shows the currently installed Keyman keyboard packages and allows you to view information about them. It enables you to download
       new keyboard packages from the website or install from local files.

   positional arguments:
       INSTALL
              download and/or install .kmp package. INSTALL can either be a downloaded .kmp file, a file:// URL pointing to a .kmp file, or a key‐
              man:// URL, possibly with a bcp47=<language> specified.

   optional arguments:
       -h, --help
              show this help message and exit

       --version
              show program's version number and exit

       -v, --verbose
              verbose logging

       -vv, --veryverbose
              very verbose logging

       -i INSTALL, --install INSTALL
              download and/or install .kmp package. INSTALL can either be a downloaded .kmp file, a file:// URL pointing to a .kmp file, or a key‐
              man:// URL, possibly with a bcp47=<language> specified.

SEE ALSO
       km-package-install(1) km-package-list-installed(1) km-package-uninstall(1)

km-config version 15.0.107-alpha (package version (local))          August 2021                                                       KM-CONFIG(1)
```

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.107-alpha](https://keyman.com/downloads/releases/alpha/15.0.107)

**Pull request merged**